docs/src/Scenario1/Evaluation_Scenario_1.md

```diff
@@ -27,7 +56,6 @@ function make_statified_model(pops; pop_assumption = (stratum, pop) -> 1)
 
     for (i, nn) in enumerate(pops)
         defs[Ns[i]] = nn
-        # ASSUMPTION: Assume only one person in each age group is infectious at the beginning of the simulation
```

Comment:
This is deleted but isn't mentioned in the text?

Reply:
It's mentioned in the doc comment. The assumption was generalized per MITRE request to allow different assumptions in question 2.

Reply:
We should probably mention that in the text then.